app/position/positionkit.cpp

```diff
@@ -105,6 +109,25 @@ AbstractPositionProvider *PositionKit::constructProvider( const QString &type, c
       QQmlEngine::setObjectOwnership( provider, QQmlEngine::CppOwnership );
       return provider;
     }
+#ifdef ANDROID
+    else if ( id == QStringLiteral( "android_fused" ) || id == QStringLiteral( "android_gps" ) )
+    {
+      bool fused = ( id == QStringLiteral( "android_fused" ) );
+      if ( fused && !AndroidPositionProvider::isFusedAvailable() )
+      {
+        // TODO: inform user + use AndroidPositionProvider::fusedErrorString() output?
```

Comment:
Todo - for later?

Reply:
Yes - we will need some updates of QML + probably how setState() in position providers works, and did not want to expand the scope of this PR too much :slightly_smiling_face: